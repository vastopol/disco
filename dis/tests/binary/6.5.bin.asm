AND R1, R1, x0
AND R4, R4, x0
ADD R4, R4, xA
LD R2, xFC
LDR R3, R2, x0
ADD R2, R2, x1
ADD R1, R1, R3
ADD R4, R4, x1F
BRp x1FB
TRAP x25
