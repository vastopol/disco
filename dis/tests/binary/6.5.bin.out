5260
5920
192A
24FC
6680
14A1
1243
193F
03FB
F025
